app/controllers/verify/review_controller.rb

```diff
@@ -23,6 +23,14 @@ def confirm_current_password
     def new
       idv_session.params.symbolize_keys!
       analytics.track_event(Analytics::IDV_REVIEW_VISIT)
+
+      phone_of_record_msg = ActionController::Base.helpers.content_tag(
+        :strong, t('idv.messages.phone.phone_of_record')
+      )
+
+      # TODO: why doesn't this correctly output html without `html_safe`?
```

Comment:
The docs say that `_html` works in views, but no controllers. So that is likely the issue: http://guides.rubyonrails.org/i18n.html#using-safe-html-translations